shared/models.go

```diff
@@ -480,6 +480,12 @@ type LegacySearchRunResult struct {
 	Passes int `json:"passes"`
 	// Total is the total number of test results for this run/file pair.
 	Total int `json:"total"`
+	// Status represents either the test status or harness status.
+	// This will be an empty string for old summaries.
+	Status string `json:"status"`
+	// NewAggProcess represents whether the summary was created with the old
+	// or new aggregation process.
+	NewAggProcess bool `json:"newAggProcess"`
```

Comment:
Quick question: do we anticipate changing the aggregation process again in the future? If so, do we want to change this to be `AggProcessVersion int`

Reply:
The hope is not to change again, and to eventually remove the old format altogether, but I can see the value in adding a variable number of formats. I have an additional focus this quarter to change it so we don't use summaries with old formats, so hopefully we'll soon be back to 1 single format.

Reply:
> Does that mean that you will be retroactively correcting old entries to use the new format?

It still needs to be investigated what is the best approach for this, but the initial thought is to invalidate the old summaries and re-aggregate the run when requested using the new method. Regenerating the old summary files would take a lot of time and resources so it's not the ideal approach. See [this comment](https://github.com/web-platform-tests/rfcs/pull/114#discussion_r905634642) for a more in-depth description.

> This means that this variable is going to be temporary and go away once we get back to a single format? If so, I would just make an issue and add it as a TODO comment here

I think that's a good idea - adding a note that the old format should ideally not be used after future changes.